NativeUI/MenuPool.cs

```diff
@@ -250,12 +251,12 @@ public void CloseAllMenus()
             }
         }
 
-        public void SetBannerType(Sprite bannerType)
+        public void SetBannerType(NativeSprite bannerType)
```

Comment:
Stuff like this is backwards incompatible. Are we sure we can't just enhance the existing classes without changing the public API?

Reply:
Maybe make the old classes a thin wrapper around Native counterparts and use them in public facing methods/properties?

Reply:
Oh yea you are right here. I'm going to add another function to keep backwards compatibility.

Reply:
Fixed on ea6088a4f2b7b923ba08c6d06a64b317c83b4d38.